scripts/add_required_checks.py

```diff
@@ -0,0 +1,50 @@
+# This script is to be used by maintainers by running it locally.
+
+from requests import put
+from os import environ
```

Comment:
Don't we run linting inside scripts?

Reply:
I don't understand your point, what do you mean?

Reply:
Black or isort would shuffle the imports here, so it looks like they're not running scripts/. No big deal.